pkg/network/protocols/common.go

```diff
@@ -6,7 +6,16 @@
 // Package protocols provides the implementation of the network tracer protocols
 package protocols
 
-import "math"
+import (
+	"fmt"
+	"io"
+	"math"
+)
+
+// WriteMapDumpHeader writes a header for a map dump
+func WriteMapDumpHeader(w io.Writer, mapName string, key interface{}, value interface{}) {
+	_, _ = io.WriteString(w, fmt.Sprintf("Map: %q, key: %T, value: %T\n", mapName, key, value))
```

Comment:
not necessary for the PR, but other things that could be useful as well: map type, key size and value size. All are exposed by `ebpf.Map

Reply:
done